src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline

```diff
@@ -459,9 +459,15 @@ sampleGetBigBook(argv.shelf);
 // [START hopper]
 // [START hopper_core]
 
-
 const library = require('@google-cloud/library').v1;
 
+/**
+ * Testing resource name overlap
+ *
+ * @param shelf {string} Test word wrapping for long lines. This is a long
+ * comment. The name of the shelf to retrieve the big book from.
```

Comment:
Sooo.... you're going to hate me for this.

But I looked across the interwebs and it looks like pretty much, across the board, the wrapping lines for method documentation are indented (2 spaces should do)

e.g.
```js
/**
 * Testing resource name overlap
 *
 * @param shelf {string} Test word wrapping for long lines. This is a long
 *   comment. The name of the shelf to retrieve the big book from.
 * @param another
```

Don't hate me. But yeah. This ^ for all the langs.

Reply:
Per our chat: out of scope for this PR. This PR only needs to take one-liners into consideration.

Reply:
Oh, cool that they're indented (I think it's more readable, as per my earlier comment above). Weird that the one or two places I looked this morning don't have the continuation line indented.

I'm +1 for doing this, and I agree that doing it in a chaser makes sense. No need to block this one.